src/api/java/com/minecolonies/api/util/constant/Constants.java

```diff
@@ -47,6 +47,7 @@
     public static final int    UPDATE_FLAG                      = 0x03;
     public static final int    TICKS_HOUR                       = TICKS_SECOND * SECONDS_A_MINUTE * SECONDS_A_MINUTE;
     public static final int    TICKS_FOURTY_MIN                 = 48000;
+    public static final int    TICKS_FIVE_MIN                   = 6000;
```

Comment:
Small nitpick if you are going to push something else before merge:
This value should be something like `TICKS_SECOND * SECONDS_A_MINUTE * 5`
You can fix the above too